tests/Doctrine/Tests/ORM/Functional/Ticket/GH6884Test.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\ORM\Event\LifecycleEventArgs;
+use Doctrine\ORM\Events;
+use Doctrine\Tests\OrmFunctionalTestCase;
+use stdClass;
+
+/**
+ * @group 6884
+ */
+final class GH6884Test extends OrmFunctionalTestCase
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->_schemaTool->createSchema(
+            [
+                $this->_em->getClassMetadata(GH6884Person::class)
+            ]
+        );
+
+        $builder = $this->getMockBuilder(stdClass::class);
+
+        $builder->setMethods(['preUpdate']);
+
+        $listener = $builder->getMock();
+
+        $listener->expects($this->atLeastOnce())->method('preUpdate');
```

Comment:
Let's be precise on how many calls of this method we expect, even if it requires us to move the expectation configuration to the test method (to make things more explicit)

Reply:
I was going to mention this, currently the method is being called 3 times but  I was expecting 2. I'm not sure why it's firing an extra time but I will add a precise count